e2e/cypress/integration/keyboard_shortcuts/ctrl_cmd_k_unreads_spec.js

```diff
@@ -108,4 +108,47 @@ describe('Keyboard Shortcuts', () => {
             cy.wrap(el).should('contain', team1Channels[1].display_name);
         });
     });
+
+    it('MM-T3002 CTRL/CMD+K - Unread Channels and input field focus', () => {
+        const team1 = teamAndChannels[0].team;
+
+        // # Visit town square channel by teamUser
+        cy.visit(`/${team1.name}/channels/town-square`);
+
+        // # Post message in other channels by otherUser
+        cy.postMessageAs({
+            sender: otherUser,
+            message: `Message on the ${teamAndChannels[0].channels[0].display_name}`,
+            channelId: teamAndChannels[0].channels[0].id,
+        }).then(() => {
+            cy.postMessageAs({
+                sender: otherUser,
+                message: `Message on the ${teamAndChannels[0].channels[1].display_name}`,
+                channelId: teamAndChannels[0].channels[1].id,
+            }).then(() => {
+                cy.postMessageAs({
+                    sender: otherUser,
+                    message: `Message on the ${teamAndChannels[0].channels[2].display_name}`,
+                    channelId: teamAndChannels[0].channels[2].id,
+                }).then(() => {
+                    // # Press keyboard shortcut for channel switcher
+                    cy.get('#post_textbox').cmdOrCtrlShortcut('k');
+
+                    // * Verify channel switcher shows up
+                    cy.get('.a11y__modal.channel-switcher').should('exist').and('be.visible').as('channelSwitcherDialog');
+
+                    // * Verify the focus is on switchers input field
+                    cy.focused().should('have.id', 'quickSwitchInput');
+
+                    // * Verify all unread channels names are showing up in the dialogs list
+                    cy.get('@channelSwitcherDialog').within(() => {
+                        // * Verify all unread channels names are showing up in the dialogs list
+                        teamAndChannels[0].channels.forEach((channel) => {
+                            cy.findByText(channel.display_name).should('be.visible');
```

Comment:
Test failed here since in this test, `otherUser` is the one currently logged in. Maybe log in as `testUser` first before visiting the test team/channel.
![Screen Shot 2021-06-01 at 10 19 25 AM](https://user-images.githubusercontent.com/5334504/120258409-90951500-c2c4-11eb-85f3-1de781150108.png)

Reply:
✅  Updated as per request/discussion